internal/watcher/watcher.go

```diff
@@ -1,4 +1,6 @@
-//go:build watcher
+//go:build !nowatcher
+
+//go:generate curl --silent https://raw.githubusercontent.com/e-dant/watcher/refs/heads/release/watcher-c/include/wtr/watcher-c.h -o watcher-c.h
```

Comment:
This makes me want to create a go package template that just tracks releases and does a `go generate` to get cgo deps (like this one). Then all the package does is allow you to manage dependencies via normal ole' go.mod files.

Reply:
We have a better option now thanks to @e-dant, I'm on it! https://github.com/e-dant/watcher/issues/58